packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/formParts/CollectingLocality/CollectingPosition.js

```diff
@@ -0,0 +1,189 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { pathBuilder } from 'coreModules/form/higherOrderComponents'
+import { Icon } from 'semantic-ui-react'
+
+const iconStyle = {
+  float: 'left',
+}
+
+const propTypes = {
+  coordinatesVerbatim: PropTypes.string,
+  georeferenceSourcesText: PropTypes.string,
+  latitude: PropTypes.string,
+  longitude: PropTypes.string,
+  maximumDepthInMeters: PropTypes.number,
+  maximumElevationInMeters: PropTypes.number,
+  minimumDepthInMeters: PropTypes.number,
+  minimumElevationInMeters: PropTypes.number,
+  onEdit: PropTypes.func,
+  uncertaintyInMeters: PropTypes.number,
+}
+
+const defaultProps = {
+  coordinatesVerbatim: undefined,
+  georeferenceSourcesText: undefined,
+  latitude: undefined,
+  longitude: undefined,
+  maximumDepthInMeters: undefined,
+  maximumElevationInMeters: undefined,
+  minimumDepthInMeters: undefined,
+  minimumElevationInMeters: undefined,
+  onEdit: undefined,
+  uncertaintyInMeters: undefined,
+}
+
+const getCoordinates = (latitude, longitude) => {
+  const latNumber = latitude && Number(latitude)
+  const lat = latNumber < 0 ? 'S' : 'N'
+
+  const lonNumber = longitude && Number(longitude)
+  const lon = longitude < 0 ? 'W' : 'E'
+
+  if (latitude && longitude) {
+    return `${Math.abs(latNumber)} ${lat}, ${Math.abs(lonNumber)} 
+      ${lon}, (WGS84) `
+  }
+  if (latitude) {
+    return `${Math.abs(latNumber)} ${lat}, (WGS84)`
+  }
+  return `${Math.abs(lonNumber)} ${lon}, (WGS84)`
+}
+
+const getElevation = (maximumElevationInMeters, minimumElevationInMeters) => {
+  if (maximumElevationInMeters === undefined) {
+    return `Elevation: ${minimumElevationInMeters} m`
+  }
+
+  if (minimumElevationInMeters === undefined) {
+    return `Elevation: ${maximumElevationInMeters} m`
+  }
+  return `Elevation: ${minimumElevationInMeters} to
+    ${maximumElevationInMeters} m`
+}
+
+const getDepth = (maximumDepthInMeters, minimumDepthInMeters) => {
+  if (maximumDepthInMeters === undefined) {
+    return `Depth: ${minimumDepthInMeters} m`
+  }
+
+  if (!minimumDepthInMeters === undefined) {
+    return `Depth: ${minimumDepthInMeters} m`
```

Comment:
remove the `!` that sneaked in here, and use maximumDepthInMeters in return value :-)